canvas_manage_course/settings/isites_migration.py

```diff
@@ -1,17 +1,32 @@
+"""
+These settings are used for:
+- isites_migration.management.commands.export_slide_tool (just exporting slide
+    tool data; uses e.g.
+- jobs put on the RQ isites_export queue (which run
+    isites_migration.jobs.migrate_files)
+"""
 from .aws import *
 
 INSTALLED_APPS = (
-    'icommons_common',
     'async',
     'django_rq',
+    'icommons_common',
     'isites_migration',
+    'lti_school_permissions',
```

Comment:
I'm sure it was, but I don't know why. This fixed it, and I didn't investigate further due to the priority of these features. I would be interested to know why, but didn't follow up, sorry. It would take me awhile to switch back over to the tlt-vagrant VM setup and re-test this, but we could pull these apps out and run the management commands with these settings to see if there's another way to get around the issue, or determine exactly what the issue is.